dawn/src/dawn/IIR/ASTStmt.cpp

```diff
@@ -110,7 +110,7 @@ std::optional<Extents> computeMaximumExtents(Stmt& stmt, const int accessID) {
       if(extents)
         extents->merge(*childExtent);
       else
-        extents = childExtent;
+        extents.swap(childExtent);
```

Comment:
why is this needed? `operator=` should work properly. In case we need `swap`, normal `swap` should do the right thing with our move semantics.